samples/gke/README.adoc

```diff
@@ -0,0 +1,96 @@
+= GKE
+
+[horizontal]
+*Language*:: JavaScript
+*Function*:: https://github.com/markfisher/sk8s/blob/master/samples/gke/slack-command/slack_command.js[slack_command.js]
+*Protocol*:: HTTP
+*Input*:: slack
+*Output*:: none (replies, implicitly)
+
+=== TL;DR
+
+If you have push access to the `sk8s` org on dockerhub, you can probably run the `test.sh` script in this
+directory and watch the whole process work. It's a bit noisy and the output may be difficult to interpret,
+however. Therefore the behaviour will be described in detail in this document.
+
+=== Deploy the FaaS
+
```

Comment:
Deploying the FaaS shouldn't be part of the sample IMO, we should have that in a separate doc and maybe link to it with a not that this example needs a LoadBalancer with a public facing IP

Reply:
I mention the deploy here since @jldec's idea was that this was an example that should run on GKE _by definition_. Reading between the lines it sounds like you're trying to relax that assumption in these docs. When I got a GKE cluster with all the defaults (except using k8s 1.8) everything worked out of the box. How exactly would you phrase the requirement that it "needs" one? Does that mean
> Your Google Cloud Platform Quotas should allow for the creation of a public-facing LoadBalancer

would be an appropriate note?

Reply:
Yes, I don't think we should limit this to GKE, as long as the user has a public facing IP and a LoadBalancer configured, it should work. 

Maybe:
"Your kubernetes cluster should allow for the creation of a LoadBalancer with a public facing IP address"

Reply:
I used a 1.8.2 GKE cluster with RBAC enabled and it still worked fine.

Reply:
OK, if you're confident making these claims, I will fully decouple the instructions and leave the details of getting and destroying a cluster in the user's hands.